pkg/kubernetes/kubernetes.go

```diff
@@ -67,6 +80,13 @@ func (k *Kubernetes) Reconcile(raw map[string]interface{}, objectspecs ...string
 		}).([]Manifest)
 	}
 
+	sort.SliceStable(out, func(i int, j int) bool {
+		if out[i].Kind() == out[j].Kind() {
+			return out[i].Name() < out[j].Name()
+		}
+		return out[i].Kind() < out[j].Kind()
```

Comment:
Super nit picky, but I prefer this pattern:

```go
		if out[i].Kind() != out[j].Kind() {
			return out[i].Kind() < out[j].Kind() 
		}
		return out[i].Name() < out[j].Name()
```

IMO it makes it easier to read, esp when sorting by more that two fields.